string2.py

```diff
@@ -37,8 +42,12 @@ def verbing(s):
 
 
 def not_bad(s):
-    # your code here
-    return
+    bad_word = 'not so bad'
+    bad_word2 = 'not that bad'
+    s = s.replace(bad_word, 'good')
+    s = s.replace(bad_word2, 'good')
```

Comment:
This is "hard-coding" a solution to pass the tests only. This solution will break very quickly if some other sentence is input to the function.